.github/workflows/heroku-merge.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy to Heroku prod on merge
+on:
+  push:
```

Comment:
I think it looks good, we can test it when we push to the release branch. I agree with Kinjal,  I think we should add 
```
branches:
      - release
```